src/services/users/DeleteAccountModal.jsx

```
@@ -0,0 +1,76 @@
+        <p className="text-sm text-gray-700 mb-4">
+          Are you sure you want to delete your account? This action is
+          irreversible.
+        </p>
+
+        {error && <p className="text-red-500 text-sm mb-2">{error}</p>}
+
+        <div className="flex justify-end gap-3">
+          <button
+            className="px-4 py-2 bg-gray-200 rounded hover:bg-gray-300"
+            onClick={() => setDeleteAccountModal(false)}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)